### ISSUE TITLE
[Edgware.RELEASE]Hystrix Dashboard Still Cannot display anything for Turbine Stream

### ISSUE THREAD

**Issue by YorkWong** [spring-cloud/spring-cloud-netflix#2637]
Although I’ve used the codes of ver. 1.4.1(fixed #2475 ), the Dashboard still cannot display the graphs normally,  it seems that the EventSource of the Dashboard cannot capture the `message` event. However, when I add a line break `\n`  (as shown below) in the codes, the Dashboard can display normally.

```java
.flatMap(data -> response.writeAndFlush(new ServerSentEvent(
                        null,
                        Unpooled.copiedBuffer("message",
                            StandardCharsets.UTF_8),
                        Unpooled.copiedBuffer(JsonUtility.mapToJson(data) + "\n",
                            StandardCharsets.UTF_8))));
```
or
```java
.flatMap(data -> response.writeAndFlush(new ServerSentEvent(
                        null,
                        Unpooled.copiedBuffer("message"+ "\n",
                            StandardCharsets.UTF_8),
                        Unpooled.copiedBuffer(JsonUtility.mapToJson(data),
                            StandardCharsets.UTF_8))));
```

**Comment by spencergibb** [comment]
I believe we already have a fix for this

**Comment by spencergibb** [comment]
can you try Edgware.SR1?

**Comment by YorkWong** [comment]
I've updated it to the Edgware.SR1, but  it seems that still not working. 
Turbine Stream
![turbine stream](https://user-images.githubusercontent.com/4627702/35090698-65bfa8ea-fc75-11e7-9c07-bb87a65ff141.png)


 Dashboard
![hystrix](https://user-images.githubusercontent.com/4627702/35090693-62d0748e-fc75-11e7-871c-e4826a9cde3d.png)

**Comment by Yenchu** [comment]
We came to the same issue with Edgware.SR1.

**Comment by eacdy** [comment]
@spencergibb @Yenchu @YorkWong @ryanjbaxter Same Issue.

**Comment by spencergibb** [comment]
@eacdy @Yenchu can you try Edgware.BUILD-SNAPSHOTS (which contain 1.4.3.BUILD-SNAPSHOT of this project). @daniellavoie can you also verify I haven't broken anything for you?

**Comment by spencergibb** [comment]
Also @YorkWong see my comment above as well.

**Comment by Yenchu** [comment]
Edgware.BUILD-SNAPSHOT works, thanks @spencergibb.

**Comment by YorkWong** [comment]
Edgware.BUILD-SNAPSHOT works, thank you so much @spencergibb.

**Comment by spencergibb** [comment]
Closed via ca80c57
